### PR TITLE
[Improvement-3471][common] JSONUtils.toMap It is not necessary to check whether the JSON method is empty again. #3471

### PR DESCRIPTION
**Pull request by zhuangchong**
## What is the purpose of the pull request

1.JSONUtils.toMap (string JSON) the JSON has been checked for null in the (string JSON) method. There is no need to call the method again to check whether the parameter is empty.

## Brief change log
1.JSONUtils.toMap Method parameters have been checked empty

```
if (StringUtils.isEmpty(json)) {
            return null;
        }
```
DataSourceService

```
        if (other != null && !"".equals(other)) {	
            Map<String, String> map = JSONUtils.toMap(other);	
            if (map.size() > 0) {
              ....
            }
        }
```

## Verify this pull request

This pull request is code cleanup without any test coverage.

---

## 拉取请求的目的是什么

1.JSONUtils.toMap（String json）方法内部已经对json校验空，没有必要再调用方法是再次校验参数是否为空。

## 简要变更日志
1.JSONUtils.toMap 方法内 参数已经校验空
```
if (StringUtils.isEmpty(json)) {
            return null;
        }
```
DataSourceService

```
        if (other != null && !"".equals(other)) {	
            Map<String, String> map = JSONUtils.toMap(other);	
            if (map.size() > 0) {
              ....
            }
        }
```

## 验证此请求

这个请求是代码清理，没有任何测试覆盖。

### CODE REVIEW

**Comment by yangyichao-mango** [comment]
Hi~
Please resolve the conflicts and resolve the sonar reported fail, thx a lot.

File: dolphinscheduler-alert/src/main/java/org/apache/dolphinscheduler/alert/utils/EnterpriseWeChatUtils.java [hunk 1]
```diff
@@ -101,11 +101,10 @@ public static String getToken() throws IOException {
             }
 
             Map<String, String> map = JSONUtils.toMap(resp);
-            if (map != null) {
-                return map.get("access_token");
-            } else {
+            if (map == null) {
                 return null;
             }
+            return map.get("access_token");
```
Comment (gabrywu) [inline dolphinscheduler-alert/src/main/java/org/apache/dolphinscheduler/alert/utils/EnterpriseWeChatUtils.java:107]: return map==null?null: map.get("access_token")

**Review by gabrywu — CHANGES_REQUESTED** [review]

**Review by CalvinKirs — APPROVED** [review]
+1

File: dolphinscheduler-alert/src/main/java/org/apache/dolphinscheduler/alert/utils/EnterpriseWeChatUtils.java (hunk 1 above)
Comment (CalvinKirs) [reply dolphinscheduler-alert/src/main/java/org/apache/dolphinscheduler/alert/utils/EnterpriseWeChatUtils.java]: please review.thx

**Review by gabrywu — APPROVED** [review]

**Pull request merged**